mobile/src/main/java/ch/epfl/sdp/mobile/ui/i18n/LocalizedStrings.kt

```diff
@@ -20,6 +20,11 @@ interface LocalizedStrings {
   val authenticateSwitchToLogIn: String
   val authenticateSwitchToRegister: String
   val authenticateErrorFailure: String
+  val authenticationBadPasswordFailure: String
+  val authenticationWrongEmailFormatFailure: String
+  val authenticationInvalidUserFailure: String
+  val authenticationIncorrectPasswordFailure: String
+  val authenticationExistingAccountFailure: String
```

Comment:
I think these strings should be prefixed with `authenticate` to remain consistent with the rest.